akamai/data/stage/base_rules.json

```diff
@@ -337,6 +337,134 @@
+                                        "options": {
+                                            "mobileDefaultChoice": "DEFAULT",
+                                            "destinationProtocol": "SAME_AS_REQUEST",
+                                            "destinationHostname": "SAME_AS_REQUEST",
+                                            "destinationPath": "OTHER",
+                                            "queryString": "APPEND",
+                                            "responseCode": 301,
+                                            "destinationPathOther": "/beta/openshift/clusters/create"
+                                        }
+                                    }
+                                ],
+                                "criteria": [
+                                    {
+                                        "name": "path",
+                                        "options": {
+                                            "matchOperator": "MATCHES_ONE_OF",
+                                            "values": [
+                                                "/beta/openshift/create*"
+                                            ],
+                                            "matchCaseSensitive": false,
+                                            "normalize": false
+                                        }
+                                    }
+                                ],
+                                "criteriaMustSatisfy": "all"
+                            },
+                            {
+                                "name": "Beta OpenShift Clusters Install",
+                                "children": [],
+                                "behaviors": [
+                                    {
+                                        "name": "redirect",
+                                        "options": {
+                                            "mobileDefaultChoice": "DEFAULT",
+                                            "destinationProtocol": "SAME_AS_REQUEST",
+                                            "destinationHostname": "SAME_AS_REQUEST",
+                                            "destinationPath": "OTHER",
+                                            "queryString": "APPEND",
+                                            "responseCode": 301,
+                                            "destinationPathOther": "/beta/openshift/clusters/install"
+                                        }
+                                    }
+                                ],
+                                "criteria": [
+                                    {
+                                        "name": "path",
+                                        "options": {
+                                            "matchOperator": "MATCHES_ONE_OF",
+                                            "values": [
+                                                "/beta/openshift/install"
```

Comment:
missing the `*` wildcard

Reply:
Thanks for catching this! I also removed the ros* redirect since it is private beta in the doc: https://docs.google.com/document/d/1NkAwv1JT22-iG-erPLMToJ3WGxadjVV8SVg-2j8WxCI

Yea, removed some old redirects like we do for the beta version. That's intentional, need Ryan Long's @ryelo input on that

Reply:
OK. I'll wait to re-review until you ping me and/or hit the button to re-request a review from me!

Reply:
Sure, thanks!